src/components/home/FirstSection.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import styled from 'styled-components';
+import { icHand, imgRoom } from 'assets';
+
+function FirstSection({ title, host }) {
+  if (title === 'unBooked') {
+    return (
+      <StyledFirstSection>
+        <StyledIcHand src={icHand} />
+        <span>아직 예약된 여행이 없습니다!</span>
+        <StyledSearchButton>숙소 검색하기</StyledSearchButton>
+      </StyledFirstSection>
+    );
+  } else {
+    return (
+      <StyledIsBooked>
+        <img src={imgRoom} />
+        <p>{title}</p>
+        <span>{host} 님이 호스팅 하는 집 전체</span>
+      </StyledIsBooked>
+    );
+  }
+}
```

Comment:
이런 경우에는 삼항 연산자 활용해서 아래처럼 쓰는 게 더 좋습니다!

```suggestion
function FirstSection({ title, host }) {
  return (
    <>
      {title === 'unBooked' ? (
        <StyledFirstSection>
          <StyledIcHand src={icHand} />
          <span>아직 예약된 여행이 없습니다!</span>
          <StyledSearchButton>숙소 검색하기</StyledSearchButton>
        </StyledFirstSection>
      ) : (
        <StyledIsBooked>
          <img src={imgRoom} />
          <p>{title}</p>
          <span>{host} 님이 호스팅 하는 집 전체</span>
        </StyledIsBooked>
      )}
    </>
  );
}
```